app.go

```diff
@@ -0,0 +1,253 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	"log"
+
+	"encoding/json"
+	"net/http"
+	_ "strconv"
+
+	"github.com/gorilla/mux"
+	_ "github.com/lib/pq"
+)
+
+type App struct {
+	Router *mux.Router
+	DB     *sql.DB
+}
+
+// tom: initial function is empty, it's filled afterwards
+// func (a *App) Initialize(user, password, dbname string) { }
+
+// tom: added "sslmode=disable" to connection string
+func (a *App) Initialize(user, password, dbname string) {
```

Comment:
don't keep everything in app.go. As we discussed, please break these down into separate packages. For ex - have a separate package for the api handler, models etc